src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -19,7 +19,8 @@
   </PropertyGroup>
 
   <ItemGroup>		
-    <PackageReference Include="librdkafka.redist" Version="0.11.3" />		
+    <PackageReference Include="librdkafka.redist" Version="0.11.3" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.4.0" />
```

Comment:
By default I'm wary of adding additional dependencies, since this opens up the possibility of versioning clashes with other libraries that reference `System.Runtime.CompilerServices.Unsage`. How much perf. benefit does using your `PtrToStructureUnsafe` provide over `PtrToStructure`? Does most of the perf. benefit you cite come from the change to `PtrToStringUTF8` (which do not require this dependency)? If so, I think we should just make those changes. If you believe i'm worried about nothing, I'd like to hear your thoughts (i'm not really an expert on dependency management).

Reply:
`Marshal.PtrToStructure` looks for a non-blittable types (like strings) and makes additional checks against struct' layout.
While `Unsafe.Read<T>` (used in `PtrToStructureUnsafe`) is a simple reinterpret_cast. The main boost (over 3x) in `Consume`/`Poll` comes exactly from it.
`PtrToStringUTF8` method' change benefit is small honestly and affects message producing.

Reply:
+ I lowered down the minimum required version of the `System.Runtime.CompilerServices.Unsafe` package.